packages/blocks/shuffle/src/shuffle.tsx

```diff
@@ -0,0 +1,88 @@
+import React, { useEffect, useReducer, useRef } from "react";
+import {
+  BlockComponent,
+  useGraphBlockService,
+} from "@blockprotocol/graph/react";
+// eslint-disable-next-line no-restricted-imports
+import Button from "@mui/material/Button";
+import Box from "@mui/material/Box";
+import { ItemList } from "./components/item-list";
+import { ActionType, initialItems, Items, shuffleReducer } from "./reducer";
+
+type BlockEntityProperties = {
+  items: Items;
+};
+
+export const Shuffle: BlockComponent<BlockEntityProperties> = ({
+  graph: {
+    blockEntity: {
+      entityId,
+      properties: { items },
+    },
+  },
+}) => {
+  const blockRootRef = useRef<HTMLDivElement>(null);
+  const { graphService } = useGraphBlockService(blockRootRef);
+
+  const [list, dispatch] = useReducer(
```

Comment:
It might be better to call `list` something like `localItems` or `draftItems` or similar, to make it clear that it's a working/draft/local copy of `items`. 

The naming at the moment doesn't make the relationship clear, and doesn't make it obvious which is the embedder-provided prop and which is the locally-managed state.

Reply:
Can you respond to this comment please @luisbettencourt? No problem if you disagree with it, but would appreciate an explanation either way

Reply:
Sorry! I missed this comment above it, it probably was behind a "load more comments" button that I didn't see.

I agree, will make this change.

Reply:
Changed to `draftItems`. e1d3e279cb160caa115af761115dca8683f15a11